wingui/pdcdisp.c

```diff
@@ -9,6 +9,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <tchar.h>
+#include <commdlg.h>
```

Comment:
What do we need this additional include for? I'm asking as this is not part of the PR issue text.